zulip_bots/zulip_bots/bots/followup/test_followup.py

```diff
@@ -34,3 +34,25 @@ def test_help_text(self) -> None:
 
         with self.mock_config_info({'stream': 'followup'}):
             self.verify_reply(request, bot_response)
+
+    def test_followup_stream(self) -> None:
+        message = dict(
+            content='foo',
+            type='stream',
+            sender_email='foo@example.com',
+        )
+        with self.mock_config_info({'stream': 'followup'}):
+            response = self.get_response(message)
+            self.assertEqual(response['content'], 'from foo@example.com: foo')
```

Comment:
It's generally a good idea to limit what you put inside a `with` statement to only those operations that require the change in context. So, stylistically, I would put the `assertEqual` statements after the `with` block.

Reply:
but if I remove the `response = self.get_response(message)` after the `with` block, I get 
`AttributeError: 'NoneType' object has no attribute 'get'` because I need to pass the config before I call `get_response()`

Reply:
@showell referred specifically to the `assertEqual` statements; the setting of the `response` variable can remain in the `with` block, but will still be accessible after the `with` is complete, so the testing of `response` can be done outside of the `with` block.

Reply:
ohh I got it, thanks a lot.